src/less/RevisionPopupWidget.less

```diff
@@ -0,0 +1,29 @@
+.ext-wwt-revisionPopupWidget-attribution {
+	margin-top: 14px;
+}
+
+/* This matches the native MediaWiki CSS for diff sizes, which may not be available on article pages. */
```

Comment:
I think if you add the `mediawiki.interface.helpers.styles` module then it'll work (and it's not a huge module). It'll also then use the `msg-brackets-start` and `msg-brackets-end` messages instead of hard-coded paretheses.

Reply:
Nice! I had no idea.